### ISSUE TITLE
Dev: fix Error: Empty Button on About page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/tdm-calculator#3040]
### Overview
We need to fix the Error: Empty Button on About so that page can be WCAG compliant.

#### Details
Category: Error
Name: Empty Button
(The Accessibility wiki for this Error has a solution)

Page that will be worked on:
About

### Action Items
- [x] Review wiki page: Accessibility, How to run the WAVE browser extension (Resource 1.01)
   - [x] Install the WAVE browser extension
- [ ] Log in as the user indicated (resource 1.02.01) 
- [ ] navigate to the page you are going to fix (resource 1.02.02 go to Edit mode) 
- [ ] Open the TDM WAVE Accessibility Audit Spreadsheet, for this Error (resource 1.03) and find the row that relates to this  issues page.
- [ ] Run the WAVE extension with no elements focused, or clicked
- [ ] Find the Error: Empty Button referenced in the Spreadsheet (1.03)
- [ ] This Error  has been resolved on other pages, so review `Project Team Solution` section (resource 1.04.01.01)
- [ ] Apply solution
- [ ] Submit PR
- [ ] Add PR to spreadsheet in PR column next to Error (resource 1.03)
- [ ] On this issue
   - [ ] Add labels: 
      - `ready for accessibility lead`
   - [ ] Change status to: `Technical Review`
   - to the epic #2892
     - [ ] Add the label `ready for product`
      - [ ] Add this comment 
         ```
          - #3040 is closed.  Please review to see if there are any remaining instances of this problem
         ```

### Resources/Instructions
- 1.01[How to run the WAVE browser extension](https://github.com/hackforla/accessibility/blob/main/docs/getting-started/wave-browser-extension.md)
- 1.02 Application details
   - 1.02.01 User Profile Required: visitor (not logged in)
   - 1.02.02 Page URL: https://tdm-dev.azurewebsites.net/about
      - Requires sign in: ~FALSE~ Admin
      - Can access page directly from URL: TRUE, Must click on 'EDIT ABOUT PAGE' button
- 1.03 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log, filter: Error: Empty button](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=1522502955)
   -  Spreadsheet Row(s): ~21~ 59
- 1.04 [Accessibility Known Issues Index](https://github.com/hackforla/accessibility/blob/main/docs/known-issues-and-solutions/solutions-index/index.md)
    - 1.04.01 [WAVE Known Accessibility Issue: Error - Empty Button](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/error-empty-button)
        - 1.04.01.01 [WAVE Known Accessibility Issue: Error - Empty Button, Project Team Solution](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/error-empty-button/#project-team-solution)

Additional reading (optional)
- https://hackforla.github.io/accessibility/

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - [Staging Deck, slide 3040](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g39673b86911_4_28#slide=id.g39673b86911_4_28)
- Release deck
   - [2026-MM-DD Release deck, slide 3040]

**Comment by Rabia2219** [comment]
@geolunalg 
Please add a level label. You can choose from one of the following:

- `good first issue`
- `level: easy`
- `level: medium`
- `level: hard`

Once you have added the level label, please:
- remove the `level: missing` label
- remove the `ready for accessibility lead` label
- add the `ready for prioritization` label

**Comment by KentKawa** [comment]
This Issue appears to have been fixed in the last accessibility audit.

<img width="1560" height="659" alt="Image" src="https://github.com/user-attachments/assets/6e7fd4c3-09ee-48ba-b933-c5812bf7debd" />

**Comment by geolunalg** [comment]
This issues was address in the initial accessability audit no PR reaquest made.

`- #3040 is closed.  Please review to see if there are any remaining instances of this problem`

**Comment by ExperimentsInHonesty** [comment]
@geolunalg @KentKawa 
I updated the spreadsheet, and am closing this issue.  Contrary to instructions below do not expand scope of this issue

>Please review to see if there are any remaining instances of this problem

**Comment by ExperimentsInHonesty** [comment]
@geolunalg please go look at the edit history of this page to find out where which pr the fix happened on.

**Comment by geolunalg** [comment]
I believe the pull request that addressed this and other issues was the pull request below by @Tony-Villa:

https://github.com/hackforla/tdm-calculator/pull/2569

**Comment by KentKawa** [comment]
@ExperimentsInHonesty @geolunalg  I believe this is the PR that fixed the issue. If it doesn't look correct could you please provide me with a screenshot that opened this issue.

**Comment by Rabia2219** [comment]
@KentKawa 
- PR #2569 is ~not~ the PR where it got fixed. ~That PR only addresses the hamburger menu.~ I see what you are referencing now and it was already shared with the stakeholders on 2025-06-24
- The following Empty Button error is also on the page, so we have revised the top part of your issue to address the image below
   - Steps to reproduce
      - Log in as Admin
      - Go to About page
      - Click 'EDIT ABOUT PAGE' button
      - Run WAVE
      - Observe all of the pencil icons are showing the Empty Button error

<img width="1909" height="946" alt="Image" src="https://github.com/user-attachments/assets/9a8cbd5c-ce92-42e3-8d7b-3e128b003e9e" />

**Comment by KentKawa** [comment]
@Rabia2219 Thank you for clearing this up.
@geolunalg It was the admin editing buttons that we discovered the other day. I have opened a PR for this here [add aria label to buttons](https://github.com/hackforla/tdm-calculator/pull/3243) that should address these errors.
